frontend/src/app/app-routing.module.ts

```diff
@@ -4,7 +4,11 @@ import { RouterModule } from '@angular/router';
 import { routes } from './routes';
 
 @NgModule({
-    imports: [RouterModule.forRoot(routes)],
+    imports: [RouterModule.forRoot(routes, {
+        // Restores previous scroll position on backward navigation.
+        // Scrolls to top on forward navigation (or anchor if provided).
+        scrollPositionRestoration: 'enabled'
```

Comment:
Users should not have to scroll to the top of the episode form manually after creating a new Episode.